backend/game/dice_handler.py

```diff
@@ -0,0 +1,25 @@
+import game.dice.dice as dice
+
+
+class DiceHandler:
+    def __init__(self):
+        self.dice_values = []
+        self.re_rolls_left = 0
+
+    def start_turn(self, starting_dice_count, re_roll_count):
```

Comment:
I don't think the dice handler should have anything regarding turns? This notion of a turn should probably be in what calls DiceHandler.

Reply:
first_roll or initial_roll probably better name

Reply:
good point